src/scripts/html-transforms.js

```diff
@@ -0,0 +1,14 @@
+/*eslint-env commonjs*/
+
+exports.removeStyleAndScript = document => {
```

Comment:
Should be using proper exports syntax:

```js
export function removeStyleAndScript (document) {
```

It should only not be if a module is shared with the server.